addon/components/denali-progress.js

```diff
@@ -4,10 +4,13 @@
  */
 import Component from '@glimmer/component';
 import { arg } from 'ember-arg-types';
-import { boolean, oneOf } from 'prop-types';
+import { boolean, number, oneOf } from 'prop-types';
 import { SIZES } from './denali-progress-enums';
 
 export default class DenaliProgressComponent extends Component {
+  @arg(number)
+  value = 0;
+
```

Comment:
Let's add back Jon's suggestion of the `arg`s as a way to document what this component accepts
```suggestion

  @arg(oneOf(COLORS))
  color = COLORS[0];

  @arg(oneOf(SHADES))
  shade = '500';
```